app/src/main/java/com/example/parkingsystemjava/mvp/view/ActivityView.java

```diff
@@ -0,0 +1,31 @@
+package com.example.parkingsystemjava.mvp.view;
```

Comment:
this should be in a base package inside the view package

Reply:
done